app/user/rest.py

```diff
@@ -491,6 +498,10 @@ def send_branding_request(user_id):
         current_app.logger.error(e)
         return jsonify({}), 400
 
+    status_code = Zendesk(contact).send_ticket()
```

Comment:
We'd need to catch the exception here like in lines [468-469](https://github.com/cds-snc/notification-api/pull/1404/files#diff-856770ce1eee6ec9509abce6fde06d0f795d88514f04c6ab6518697a9e3bcc08R468).